e2e/cypress/tests/integration/collapsed_reply_threads/root_post_spec.js

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Stage: @prod
+// Group: @collapsed_reply_threads
+
+describe('Collapsed Reply Threads', () => {
+    let testTeam;
+    let testUser;
+    let otherUser;
+    let testChannel;
+    let rootPost;
+
+    before(() => {
+        cy.apiUpdateConfig({
+            ServiceSettings: {
+                ThreadAutoFollow: true,
+                CollapsedThreads: 'default_off',
+            },
+        });
+
+        // # Create new channel and other user, and add other user to channel
+        cy.apiInitSetup({loginAfter: true, promoteNewUserAsAdmin: true}).then(({team, channel, user}) => {
+            testTeam = team;
+            testUser = user;
+            testChannel = channel;
+
+            cy.apiSaveCRTPreference(testUser.id, 'on');
+            cy.apiCreateUser({prefix: 'other'}).then(({user: user1}) => {
+                otherUser = user1;
+
+                cy.apiAddUserToTeam(testTeam.id, otherUser.id).then(() => {
+                    cy.apiAddUserToChannel(testChannel.id, otherUser.id);
+
+                    // # Post a message as other user
+                    cy.postMessageAs({sender: otherUser, message: 'Root post: Delete test', channelId: testChannel.id}).then((post) => {
+                        rootPost = post;
+                    });
+                });
+            });
+        });
+    });
+
+    beforeEach(() => {
+        // # Visit the channel
+        cy.visit(`/${testTeam.name}/channels/${testChannel.name}`);
+    });
+
+    it('MM-T4445 thread is removed when root post is deleted (current behavior is incorrect, see comments below)', () => {
```

Comment:
Missed to mention but I suggests to match the title with the Zephyr. SO either change here or update the title in Zephyr.
```suggestion
    it('MM-T4445 CRT - Delete root post', () => {
```